ktx/src/main/java/permissions/dispatcher/ktx/PermissionRequestViewModel.kt

```diff
@@ -27,14 +27,14 @@ internal class PermissionRequestViewModel : ViewModel() {
         onPermissionDenied: WeakReference<Fun>?,
         onNeverAskAgain: WeakReference<Fun>?
     ) {
-        permissionRequestResult.observe(owner, {
+        permissionRequestResult.observe(owner) {
```

Comment:
This is not main topic of this PR but fixed warning about trailing lambda syntax